spec/features/user_views_lots_of_reviews_via_pagination_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+feature 'user views lots of reviews via pagination' do
+
+# As a user
+# I want to view lots of reviews
+# without all books being loaded ahead of time
+#
+# Acceptance Criteria:
+# [x] When visiting the root or books index page, a user should see a list of
+#     20 reviews.
+# [x] When a user clicks 'More Reviews', 20 more reviews are loaded.
+# [x] This feature should function correctly before JS functionality is added
+# [ ] JS Functionality: When the user scrolls to the bottom of the page,
+#     more books are loaded automatically without a page refresh and the
+#     'more reviews' button goes away.
+# [x] Pagination should be handled by the Kaminari gem
+
+  let!(:dynamic_book) do
+    b = FactoryGirl.build(:dynamic_book)
+    b.author = FactoryGirl.create(:author)
+    b.user = FactoryGirl.create(:user)
+    b.genre = Genre.create(genre_name: "Mystery")
+    b.save
+    b
```

Comment:
This is not the way to create associations in FactoryGirl. An abbreviated example: you can do something like:
```
let!(:author) { FactoryGirl.create(:author)
let!(:dynamic_book) do
  FactoryGirl.create(:dynamic_book, author: author)
end
```

Which would create a `Book` with an association to the `Author` that you created.